Code/Editor/Plugins/FFMPEGPlugin/main.cpp

```diff
@@ -24,11 +24,7 @@ PLUGIN_API IPlugin* CreatePluginInstance(PLUGIN_INIT_PARAM* pInitParam)
     // Make sure the ffmpeg command can be executed before registering the command
     if (!CFFMPEGPlugin::RuntimeTest())
     {
-        AZStd::string msg =
-            "FFMPEG plugin: Failed to execute FFmepg. Please run Setup Assistant, "
-            "go to the 'Optional software' section of the 'Install software' tab, "
-            "and make sure the FFmpeg executable is correctly configured.";
-        GetIEditor()->GetSystem()->GetILog()->Log(msg.c_str());
+        GetIEditor()->GetSystem()->GetILog()->Log("FFMPEG plugin: Failed to execute FFmepg. Please install FFmepg.");
```

Comment:
Spelling:  'FFmepg'

Reply:
We should be removing the FFMPEGPlugin in general.
I believe @bosnichd is planning to remove to it?

Reply:
sorry, not following what the right spelling should be. The original message had `FFmpeg` and `FFMPEG` for the plugin name.

Reply:
"Please install FFmepg"

Reply:
Also "Failed to execute FFmepg"

Reply:
If we nuke it, we don't have to spell it correctly

Reply:
will leave the nuking for another PR

Reply:
Cool with me.